c_emulator/rv_cfg_func.h

```diff
@@ -0,0 +1,85 @@
+// vim: set tabstop=4 shiftwidth=4 expandtab
+// ============================================================================
+// Filename:    rv_cfg_func.h
+//
+// Description: Functions prototype support for rv_cfg
+//
+// Author(s):   Bill McSpadden (bill@riscv.org)
+//
+// Revision:    See git log
+// ============================================================================
+
+#pragma once
+
+#include "sail.h" 
+#include <libfyaml.h>
+#include <sys/stat.h>
+#include "riscv_platform_impl.h"
+
+//This macro must be defined before including pcre2.h. For a program that uses
+//only one code unit width, it makes it possible to use generic function names
+//such as pcre2_compile().
+#define PCRE2_CODE_UNIT_WIDTH 8
```

Comment:
PCRE2 is an implementation detail so this include should move to the source file.